languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,54 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_QUE" name="quelle que" default="temp_off">
```

Comment:
This is what I've seen in the RE:

- Est-ce que qu'elle qu'un peut m'aider ?

I think this is worth a subrule correcting that directly to "quelqu'un", what do you think?